src/pages/FavoritesPage/FavoritesPage.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components/macro'
+import PodcastCard from '../../components/PodcastCard/PodcastCard'
+
+export default function FavoritesPage({
+  favoritePodcasts,
+  onClickDetails,
+  onClickBackFavorites,
+}) {
+  return (
+    <PageContainer>
+      <button onClick={onClickBackFavorites}>Back</button>
```

Comment:
I'd recommend using `Link` or `NavLink` from `react-router-dom`. You can style it as a button and don't need the functions...
```suggestion
      <Link to="/">Back</Link>
```